scripts/api.js

```diff
@@ -183,3 +183,43 @@ api.getCSS = function (url, callback) {
 		error: errorHandler,
 	});
 };
+
+/**
+ *
+ * @param {string} url
+ * @param {APISuccessCB} callback
+ * @returns {void}
+ */
+api.getJS = function (url, callback) {
```

Comment:
Isn't this function a literal copy past of `api.getCSS` ? 
If so, then this is duplicate code, you can just rename `getCSS` into something like `getRawContent` and replace at the appropriate places in the code and it should work pretty nicely.